src/configure/auto_mozilla.rs

```diff
@@ -217,3 +234,84 @@ fn moz_autoconfigure_starttag_cb<B: std::io::BufRead>(
         moz_ac.tag_config = MozConfigTag::Username;
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_parse_outlook_autoconfig() {
+        let xml_raw =
```

Comment:
can you add a comment how this data was retrieved?

Reply:
Fixed in the second commit